src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Program.cs

```diff
@@ -1,47 +1,51 @@
-﻿#if BlazorWebAssembly
+﻿using System.Threading.Tasks;
+#if BlazorWebAssembly
+using System.Net.Http;
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 using Microsoft.Extensions.DependencyInjection;
-using System.Net.Http;
 #elif BlazorServer
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Hosting;
+using Microsoft.AspNetCore.Builder;
 #endif
-using System;
-using System.Threading.Tasks;
 
-namespace Bit.Client.Web.BlazorUI.Playground.Web
+namespace Bit.Client.Web.BlazorUI.Playground.Web;
+
+public class Program
 {
-    public class Program
-    {
-        public static async Task Main(string[] args)
-        {
 #if BlazorWebAssembly || BlazorServer
```

Comment:
Could you please use one Main method to be the same as TodoTemplate's web project?
Also add UseUrls

https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Program.cs#L12-L20

https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Program.cs#L40-L42

@RezaKargar